Styles/wishlist.css

```diff
@@ -0,0 +1,41 @@
+.wishWrap {
+  background-color: var(--blue-gray);
+  overflow: auto;
+  height: calc(100vh - 4.8rem);
+}
+.wishWrap h1 {
+  color: var(--snow);
+}
+.wishStrip {
+  color: var(--darc);
+  background-color: var(--snow);
+  display: grid;
+  grid-template-columns:
+    4rem minmax(13rem, 1fr)
+    minmax(10rem, 1fr) 11rem minmax(10rem, 0.6fr);
+  grid-template-rows: 5rem;
+  gap: 1rem;
+}
+.wishContainer .wishStrip {
+  grid-template-rows: 10rem;
+}
+.wishStrip h3 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
Can these be imported from the component library?

Reply:
i did not create a utility class for that since i didn't use this feature that much in CL, will add it when i refactor the component library